cluster/addons/dns/coredns/coredns.yaml.base

```diff
@@ -143,6 +143,11 @@ spec:
             path: /health
             port: 8080
             scheme: HTTP
+        readinessProbe:
+          httpGet:
+            path: /health
+            port: 8080
+            scheme: HTTP
           initialDelaySeconds: 60
```

Comment:
These settings (e.g. `initialDelaySeconds`, `successThreshold`) were for livenessProbe. We would probably like to keep them and add another set of settings for readinessProbe?

Reply:
oops, yes, those settings were meant to remain for `livenessProbe`
I'll update.